src/GovUk.Education.ExploreEducationStatistics.Common/Utils/GeographicLevelUtils.cs

```diff
@@ -8,8 +8,11 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Utils;
 
 public static class GeographicLevelUtils
 {
-    private static readonly Lazy<GeographicLevel[]> LevelsLazy =
-        new(EnumUtil.GetEnumsArray<GeographicLevel>);
+    private static readonly Lazy<IReadOnlyList<GeographicLevel>> LevelsLazy =
+        new(EnumUtil.GetEnums<GeographicLevel>());
+
+    private static readonly Lazy<IReadOnlyList<GeographicLevel>> OrderedLevelsLazy =
```

Comment:
do we want to add some extra lazy's just for the list of values and list of labels? Utilising your new cached `GetEnumValues<TEnum>` and `GetEnumLabels<TEnum>` `EnumUtil` methods. Then these cached values can be used in the schema filter

Reply:
I've actually just gone in another direction and just scrapped the cached orderings completely. On reflection, it's not really necessary and it just creates a bunch more code we don't really need.

Just changed this to order within the schema filter instead.